src/instruction_graph_generator.cc

```diff
@@ -1547,7 +1547,8 @@ instruction* generator_impl::launch_task_kernel(batch& command_batch, const exec
 		assert(chunk.execution_range.get_area() > 0);
 		assert(chunk.device_id.has_value());
 		return create<device_kernel_instruction>(command_batch, *chunk.device_id, tsk.get_launcher<device_kernel_launcher>(), chunk.execution_range,
-		    std::move(allocation_map), std::move(reduction_map) CELERITY_DETAIL_IF_ACCESSOR_BOUNDARY_CHECK(, tsk.get_id(), tsk.get_debug_name()),
+		    std::move(allocation_map),
+		    std::move(reduction_map) CELERITY_DETAIL_IF_ACCESSOR_BOUNDARY_CHECK(, tsk.get_type(), tsk.get_id(), tsk.get_debug_name()),
```

Comment:
Nitpick: can we have the OOB-check-specific stuff on a separate line?

Reply:
We can if I do an explicit `#if CELERITY_ACCESSOR_BOUNDARY_CHECK ... #endif` instead of the wrapping macro. `clang-format` does not want to play ball if I force a newline, it gets confused by the `,` being inside of the macro parentheses.